apex/normalization/instance_norm.py

```diff
@@ -0,0 +1,808 @@
+from enum import Enum
+from typing import Any, Dict, List, Optional, Tuple
+
+import torch
+from torch.nn.modules.batchnorm import _NormBase
+
+from nvfuser._C import DataType, Fusion, FusionDefinition, Scalar, Tensor
+
+
+NamedAxis = Enum("NamedAxis", ["BATCH", "CHANNEL"])
+
+
+def torch2datatype(dt: torch.dtype) -> Optional[DataType]:
```

Comment:
Let's expose `aten_to_data_dtype` defined in `third_party/nvfuser/csrc/type.h` instead of wrapping it in python again.

Also, why is there an entry for `bool`, if we are only looking at torch.ScalarType, it should always be a `torch.bool` instead?!

Reply:
Removed the `bool` entry which as you mentioned has no place there. As for exposing `aten_to_data_dtype` I will do this and probably rename it as it will take pytorch as opposed to aten types.